rfcs/0001-bluemoji.md

```diff
@@ -119,11 +120,7 @@ or if ImgProxy adds support for APNG animations. See
 
 - **Accessibility concerns:** The colon-wrapped alias provides a slight amount
   of contextual information so it has been recommended as the fallback text, but
-  there's no way of enforcing that given how ATProto facets work. A suggested
-  fallback character (`◌`) has been provided to provide guide-rails for an
-  approach that may become popular as more clients begin to support Bluemoji in
-  the future, especially given that the facet contains more accessibility
-  information than the colon-wrapped alias provides.
+  there's no way of enforcing that given how ATProto facets work.
```

Comment:
Removing the non-descriptive fallback from previous revisions here seems good to me 👍

The full fallback may be a bit annoying with the low character limit in the protocol, but that's app-specific to Bluesky, so I agree it's best not to suggest a less useful alternative.